step-templates/octopus-register-listening-target.json

```diff
@@ -1,135 +1,135 @@
+  },
+  "Parameters": [
+    {
+      "Id": "f4475bac-5fdb-4d3b-a5de-20dafb8d847e",
```

Comment:
This looks like a new ID, did you paste in the entire JSON body or just the script?